homeassistant/components/light/mochad.py

```diff
@@ -67,8 +70,15 @@ def brightness(self):
 
     def _get_device_status(self):
         """Get the status of the light from mochad."""
-        with mochad.REQ_LOCK:
-            status = self.light.get_status().rstrip()
+        from pymochad import device
+
+        if self._controller.ctrl and self._controller.connect_event.wait():
```

Comment:
Why is it better to move the `wait` call to here? Isn't that part of the set up phase? It would be best to change the library so that we don't need to wait at all. I suggested a change like that earlier. But otherwise it should be part of the set up phase in the component.